### DISCUSSION TITLE
Incorrect URL generated if assets are published and config('livewire.asset_url') is defined using env('ASSET_URL')

### DISCUSSION THREAD

**Discussion by pooley182** [livewire/livewire#8210] [7. Bugs (archived)]
### Livewire version

v3.4.9

### Laravel version

v10.48.4

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Publish your vendor assets with ` php artisan vendor:publish --force --tag=livewire:assets`
2. (Optional) Upload your `/public` assets to S3.
3. Publish `config/livewire.php` and set `asset_url` to the URL for your published assets, in my case I used the S3 bucket from step 2.
4. open your site, see that livewire.min.js fails to load and notice that the URL generated by `@LivewireScripts` is purely `asset_url` from your config file without the path to the asset (i.e.  /vendor/livewire/livewire.min.js)

Alternatively check out the linked repository where I have completed these steps

### Repository or Wirebox URL

https://github.com/pooley182/livewire-asset-url-bug

### Additional information

This seems to be caused by [this piece of code](https://github.com/livewire/livewire/blob/main/src/Mechanisms/FrontendAssets/FrontendAssets.php#L230) which blindly takes asset_url and doesn't attach any paths to the expected asset.

asset_url isn't defined in the default `config/livewire.php` and  isn't documented how to set it in the livewire docs.
Because it shares it's name with laravels asset_url that is used by the asset() helper method it's not clear that this should fe a full path to a file.

I think all that's needed is some clarity in the docs that it should be set to a full path directly to the livewire.js file
e.g. `'asset_url' => env('ASSET_URL' ,  'http://localhost') . '/vendor/livewire/livewire.min.js',`

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@pooley182 yeah that is confusing. I think putting the full path in `asset_url` makes sense. PRs to update the docs are welcome!
